sample/app/src/main/java/com/example/activitystarter/fragment/TabbedFragmentActivity.java

```diff
@@ -24,7 +25,8 @@ protected void onCreate(Bundle savedInstanceState) {
         mViewPager.setAdapter(mSectionsPagerAdapter);
     }
 
-    public class SectionsPagerAdapter extends FragmentPagerAdapter {
+    public class SectionsPagerAdapter extends FragmentPagerAdapter
+    {
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html